docs/install-and-setup.md

```diff
@@ -91,15 +91,13 @@ should never need or want to edit these.
 ## Activate API for the script
 
 Now that the script is saved in your Google Drive folder we need to activate it.
-To do so click the menu `Resources->Advanced Google services`.  
+To do so click the menu `Resources->Advanced Google services`.
 In the popup which will open set "Calendar API" to `enabled` (click the switch
-on its row on the right) and press "Okay".  
-Once you have done this open `Resources->Advanced Google services` again and
-click on the link which says "Google Cloud Platform API Dashboard": you will be
-taken to another page. Click on "Library" in the menu on the left, search for
-"Google Calendar API" in the search box and open it. Now click `Enable` and
-close this page. There is no need to create any credentials as suggested on the
-page.  
+on its row on the right) and press "Okay".
+Once you have done this, open [Google Cloud Platform API Dashboard][https://console.cloud.google.com/apis/dashboard],
```

Comment:
following this link works only if you have only one script using google apis. even tho ppl who have more of them should know how to check the correct one it should be mentioned here (changing the project in the dropdown menu on the top). on the other hand the old method to follow the link from the menu box works for everyone and always open the correct project

Reply:
i see you mention that you dont see Google Cloud Platform API Dashboard in Resources->Advanced Google services.

Mine looks like that and certainly there is correct link:
![Screenshot_20191230_103952](https://user-images.githubusercontent.com/6292364/71576374-c3941180-2af0-11ea-98df-827b21369cf5.png)

Can someone confirm that it's indeed missing to some?

Reply:
![image](https://user-images.githubusercontent.com/3449343/71576765-f8ed2f00-2af1-11ea-8097-934bace03f32.png)

Reply:
Another user reached out to me and confirmed that the link is not there for him, so this is definitely something we should address.

I'm with @baatochan on the additional step of the dropdown menu and here's my suggestion:
```suggestion
Once you have done this, open [Google Cloud Platform API Dashboard][https://console.cloud.google.com/apis/dashboard],
click on the dropdown menu left at the top, next to "Google Cloud Platform"
and choose your Google Contacts Events Notifier project from the list, then
```